ios/ble/central/CentralManagerDelegate.swift

```diff
@@ -12,8 +12,8 @@ extension Central {
             let scanResponseData = dataDict?[CBUUID(string: "AB2A")]  as! Data
             let advertisementData = dataDict?[CBUUID(string: "AB29")]  as! Data
             let publicKeyData =  advertisementData.subdata(in: advertisementData.count-5..<advertisementData.count) + scanResponseData
-            if Wallet.shared.isSameAdvIdentifier(advertisementPayload: advertisementData) {
-                Wallet.shared.setVerifierPublicKey(publicKeyData: publicKeyData)
+            print("veri pub key::", publicKeyData)
+            walletDelegate?.hasSameIdentifier(payload: advertisementData, publicData: publicKeyData) {
```

Comment:
This line is missed out here

`Wallet.shared.setVerifierPublicKey(publicKeyData: publicKeyData)`

Reply:
Moved the condittion check also inside the delegate method for more cleaner approach.